src/Core/MenuBuilder/MenuManager.php

```diff
@@ -4,20 +4,23 @@
 
 namespace Forumify\Core\MenuBuilder;
 
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 use Twig\Extension\RuntimeExtensionInterface;
 
 abstract class MenuManager implements RuntimeExtensionInterface
 {
     /** @var array<MenuBuilderInterface> */
     protected array $menuBuilders;
+    private AuthorizationCheckerInterface $authorizationChecker;
 
-    public function __construct(iterable $menuBuilders)
+    public function __construct(iterable $menuBuilders, AuthorizationCheckerInterface $authorizationChecker)
```

Comment:
Use property promotion and inject `Security` instead of `AuthorizationCheckerInterface`